nuheat/config.py

```diff
@@ -1,15 +1,12 @@
-API_URL = "https://www.mynuheat.com/api"
-
-AUTH_URL = API_URL + "/authenticate/user"
-THERMOSTAT_URL = API_URL + "/thermostat"
-
-REQUEST_HEADERS = {
-    "Content-Type": "application/x-www-form-urlencoded",
-    "Accept": "application/json",
-    "Host": "mynuheat.com",
-    "DNT": "1",
-    "Origin": "https://mynuheat.com/api"
+BRANDS = ("NUHEAT", "MAPEHEAT")
```

Comment:
it'd be nice if we defined these individual strings as constants as well so that we can use the constants (at least the default, "NUHEAT" one) in code, rather than strings.